CGCharter-1727386911.html

```diff
@@ -241,15 +248,37 @@ <h2 id="decision">
       ahead with.
     </p>
     <p>
-      Any decisions reached at any meeting are tentative and should be recorded
-      in a GitHub Issue for groups that use GitHub and otherwise on the group's
-      public mail list. Any group participant may object to a decision reached
-      at an online or in-person meeting within 7 days of publication of the
-      decision provided that they include clear technical reasons for their
-      objection. The Chairs will facilitate discussion to try to resolve the
-      objection according to this decision process.
+      To afford asynchronous decisions and organizational deliberation, any 
+      resolution (including publication decisions) taken in a face-to-face 
+      meeting or teleconference will be considered provisional.
+      A call for consensus (CfC) will be issued for all resolutions via 
+      email to public-swicg@w3.org, preferably with the abbreviation CfC early
+      in the subject line to catch more attention.
+    </p>
+    <p>
+      Any group participant may object to a 
+      decision reached at an online or in-person meeting within 14 days of 
+      publication of the decision, provided that they include 
+      clear reasons for their objection grounded in the scope and documented
+      goals of the CG and its work items.
+    </p>
+    <h3 id="calls-for-consensus">
+      Calls for Consensus
+    </h3>
+    <p>  
+      The presence of formal resolutions will be indicated by a "CFC" prefix in
```

Comment:
Elsewhere in the document you use `CfC`, here it's `CFC`
```suggestion
      The presence of formal resolutions will be indicated by a "CfC" prefix in
```